src/backend/expungeservice/endpoints/auth.py

```diff
@@ -31,7 +31,7 @@ def wrapper(*args, **kwargs):
             if auth_hdr == None:
                 return 'Missing Authorization header!', 401
 
-            match = re.match('Bearer\s+(.+)', auth_hdr)
+            match = re.match('Bearer +(.+)', auth_hdr)
```

Comment:
Why do we need to support a variable number of spaces after "Bearer" in the header? If this is something we need to do, it should be commented.

This is a simple regex so it's probably fine but just as a style issue I'd vote for replacing the regex with string methods instead.

Reply:
I'm with Max. I suggest splitting on whitespace or 'Bearer' and checking the second item in the list.

Reply:
Yep I was sloppy with this, replacing it with split() and strip() string ops make sense.